src/content/community/get-involved/open-research/index.md

```diff
@@ -0,0 +1,240 @@
+
+### Layer 2
+
+There are now several Layer 2 protocols that scale Ethereum using different techniques for batching transactions and securing them on Ethereum layer 1. This is a very rapidly growing topic with a lot of research and development potential.
+
+#### Links
+
+[Our Layer-2 page](https://ethereum.org/en/layer-2/)
+
+[EthHub Layer-2](https://docs.ethhub.io/ethereum-roadmap/layer-2-scaling/zk-rollups/)
+
+[ethresear.ch Layer 2](https://ethresear.ch/c/layer-2/32)
+
+
+### Bridges
+
+One particular area of Layer-2 that requires more research and development is safe and performant bridges. This includes bridges between various Layer-2s and bridges between Layer 1 and Layer 2. This is a particularly important area of researchg because bridges are commonly targeted by hackers.
+
+#### Links
+
+[Our Bridges page](https://ethereum.org/en/developers/docs/bridges)
+
+[Vitalik on bridges](https://old.reddit.com/r/ethereum/comments/rwojtk/ama_we_are_the_efs_research_team_pt_7_07_january/hrngyk8/)
+
+
+### Sharding
+
+Sharding Ethereum's blockchain has long been part of the development roadmap. However, new scaling solutions such as "Danksharding" are currently taking center stage. 
+
+#### Links
+
+[Our Sharding page](https://ethereum.org/en/develoeprs/docs/shard-chains)
+
+[Proto-Danksharding notes](https://notes.ethereum.org/@vbuterin/proto_danksharding_faq)
+
+[Bankless Danksharding video](https://www.youtube.com/watch?v=N5p0TB77flM)
+
+### Hardware
+
+Running nodes (see our [nodes page](https://ethereum.org/en/developers/docs/nodes-and-clients/run-a-node)) on modest hardware is fundamental to keeping Ethereum decentralized. Therefore, active research into minimizing the hardware requirements to run nodes is an important area of research.
+
+#### Links
+
+[Ethereum on ARM](https://ethereum-on-arm-documentation.readthedocs.io/en/latest/)
+
+[Ethresear.ch Scaling](https://ethresear.ch/c/sharding/6)
+
+[Ethresear.ch Layer 2](https://ethresear.ch/c/layer-2/32)
+
+[Ethereum Sharding Research Compendium](https://notes.ethereum.org/@serenity/H1PGqDhpm?type=view)
+
+[Danksharding (Polynya)](https://polynya.medium.com/danksharding-36dc0c8067fe)
+
+## Security
+
+Security is a broad topic that might include spam/scam prevention, wallet security, hardware security, crypto-economic security, bug hunting and testing of applications and client software and key-management. Contributing to knowledge in these areas will help stimulate mainstream adoption.
+
+### Cryptography & ZKP
+
+Zero-knowledge proofs (ZKP) and cryptography are critical for building privacy and security into Ethereum and its applications. Zero-knowledge is a relatively young but fast-moving space with many open research and development opportunities.
+
+#### Links
+
+[0xparc blog](https://0xparc.org/blog)
+
+[Ethresear.ch ZK](https://ethresear.ch/c/zk-s-nt-arks/13)
+
+[zkp.science](https://zkp.science/)
+
+[Zero Knowledge podcast](https://zeroknowledge.fm/)
+
+
+### Wallets
```

Comment:
I think account abstraction is a good research area to touch in the wallets section, e.g.
- https://eips.ethereum.org/EIPS/eip-2938
- https://eips.ethereum.org/EIPS/eip-4337